CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -307,10 +327,12 @@ function PlayerHospital:msgMultiReceptionDesks()
 end
 
 --! Give advice to the user about maintenance of plants.
-function PlayerHospital:msgPlant()
+--!param placed (bool) If a plant was placed
+function PlayerHospital:msgPlant(placed)
```

Comment:
Function name doesn't cover what it actually does.

Reply:
Probably not me, but any suggestions? We have several of these functions with this naming scheme currently, so maybe it isn't worth changing just this one right now?

Reply:
advisePlantMaintenance or alternately warnPlantMaintenance

The argument can be assumePlants
```lua
local hasPlants = assumePlants or (self:countPlants() > 1)```

Reply:
I've gone with Alberth's suggested alternative and renamed it `advisePlants`